src/main/connector/hardware/MIB2Solaris/MIB2Solaris.yaml

```diff
@@ -0,0 +1,55 @@
+---
+extends:
+- ../Hardware/Hardware
+- ../MIB2-header/MIB2-header
+- ../SunEthernetDriver-header/SunEthernetDriver-header

```

Comment:
`../SunEthernetDriver-header/SunEthernetDriver-header` is not present.

Reply:
MIB2Solaris should be removed (moved back to Enterprise)